generatebundlefile/ecr_helper.go

```diff
@@ -5,8 +5,9 @@ import (
 	"os/exec"
 	"strings"
 
-	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
 	"github.com/pkg/errors"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
```

Comment:
Can we have a better name for the import?

Reply:
This was fixed in another pr